packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -190,6 +190,13 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
     private _disposed = false;
     public get disposed() { return this._disposed; }
 
+    /**
+     * Tombstone is a temporary feature that marks a datastore as deleted, although it does not actually delete the
```

Comment:
I would not say marks as deleted. And this should also say what tombstone means for a data store - this data store cannot send / receive ops, signals and it cannot be loaded.